go/adbc/driver/snowflake/snowflake_database.go

```diff
@@ -176,6 +176,7 @@ func (d *databaseImpl) SetOptions(cnOptions map[string]string) error {
 	// set default application name to track
 	// unless user overrides it
 	d.cfg.Application = d.defaultAppName
+	d.cfg.ValidateDefaultParameters = gosnowflake.ConfigBoolFalse
```

Comment:
Please move this to the function that initializes the databaseImpl struct so it's always set no matter the path made by the options into this class.

And also add a comment explaining why this achieves. Not in terms of dbt Core but something about this allowing connections to be created for objects that don't exist in the db yet.

Reply:
updated